_posts/kubernetes/2018-07-19-Deciding-on-Manifest-Namespaces.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2018-07-19
+title: Deciding on Manifest Namespaces
+categories:
+   - Kubernetes
+description: Q&A about manifest namespaces
+type: Document
+---
+
+### Question:
+
+*This is a Q&A pulled from the Spinnaker Community Slack team, which [we encourage you to join here](http://join.spinnaker.io).*
+
+How does the provider decide which namespace to use to apply manifests? Just tried to apply my first one and it created it on the spinnaker namespace. Is that the default?
```

Comment:
let's wrap spinnaker in backticks so that is shows up like `spinnaker`.

Reply:
backtick being which key, exactly?  ' / ` other?  new to me :)

what's the rule about apply this in the future?  Anytime someone uses "spinnaker namespace"?  or overtime spinnaker is referenced?